src/features/defi/providers/osmosis/components/OsmosisManager/Lp/Withdraw/LpWithdrawCommon.ts

```diff
@@ -7,9 +7,11 @@ type EstimatedFee = {
 }
 
 type WithdrawValues = {
-  underlyingAsset0: OsmosisToken
-  underlyingAsset1: OsmosisToken
+  underlyingAsset0: OsmosisToken & { amountCryptoHuman: string }
+  underlyingAsset1: OsmosisToken & { amountCryptoHuman: string }
   shareInAmountBaseUnit: string
+  amountCryptoHuman: string
+  fiatAmount: string
```

Comment:
Ditto should be able to derive the newly added crypto human amounts and fiat amount